opensuse_buildservice/qmail-send.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
```

Comment:
This can now go away, the repository already has such a file. ```make qmail-send.service``` will create it.

Reply:
OK